lib/pengine/unpack.c

```diff
@@ -468,6 +476,13 @@ expand_remote_rsc_meta(xmlNode *xml_obj, xmlNode *parent, pe_working_set_t *data
         crm_xml_add(attr, XML_NVPAIR_ATTR_VALUE, remote_allow_migrate);
     }
 
+    if (container_managed) {
+        attr = create_xml_node(xml_tmp, XML_CIB_TAG_NVPAIR);
+        crm_xml_set_id(attr, "%s_%s", remote_name, "meta-attributes-managed");
+        crm_xml_add(attr, XML_NVPAIR_ATTR_NAME, XML_RSC_ATTR_MANAGED);
+        crm_xml_add(attr, XML_NVPAIR_ATTR_VALUE, container_managed);
+    }
+
```

Comment:
Doh! I just noticed we have a copy-and-paste error in the block above this. Can you add a commit to change "meta-attributes-container" to "meta-attributes-allow-migrate"? As it is, I think it would fail validation, and the only reason we don't have a problem is that no one sets allow-migrate explicitly on remote connection resources.